.github/workflows/ci.yml

```diff
@@ -8,9 +8,10 @@ on: [push, pull_request]
 jobs:
   java-ci-with-maven-adopt:
     runs-on: ubuntu-latest
-    
+
     steps:
-    - run: | 
+    - uses: actions/checkout@v2
+    - run: |
```

Comment:
Add a name for this step. For example: `build azure-sdk-for-java feature/spring-boot-3 branch`